Formula/go.rb

```diff
@@ -5,6 +5,7 @@ class Go < Formula
   mirror "https://fossies.org/linux/misc/go1.20.2.src.tar.gz"
   sha256 "4d0e2850d197b4ddad3bdb0196300179d095bb3aefd4dfbc3b36702c3728f8ab"
   license "BSD-3-Clause"
+  revision 1
```

Comment:
Does this affect enough users to be worth a revision bump?

Reply:
Maybe it's not, I just bumped it since we're technically changing what's delivered in the bottle. Open to not bumping the revision here.

Reply:
Let's just skip it for now and bump the revision if more users complain about something that this change would have fixed.